beta/docker/README.md

```diff
@@ -48,41 +53,78 @@ All following steps should be run on the same computer where you are already usi
     ```
 
 3. Save the `scimsession` credentials file from [the Automated User Provisioning setup](https://start.1password.com/integrations/directory/) to this working directory.
+   > **Note**
+   >
+   > 💻 If you saved your `scimsession` file as an item in your 1Password account, you can
+   > [use 1Password CLI](https://developer.1password.com/docs/cli/reference/commands/read) to save the file to this
+   > directory. For example:
+   >
+   > ```sh
+   > op read "op://Private/scimsession file/scimsession" --out-file ./scimsession
+   > ```
+
+4. Open `scim.env` in your favourite text editor. Set the value of `OP_TLS_DOMAIN` to the fully qualififed domain name of the public DNS record for your SCIM bridge created in [Get started](#get-started). For example:
+
+   ```dotenv
+   # ...
+   OP_TLS_DOMAIN=scim.example.com
+   
+   # ...
+   ```
+
+   Save the file.
 
-4. Create and use a Docker context:
+5. Create a Docker context to use for connecting to the Linux server:
 
-   *Example commands:*
+   *Example command:*
 
    ```sh
    docker context create op-scim-bridge \
        --description "1Password SCIM Bridge Docker host" \
        --docker host=ssh://user@scim.example.com
-   docker context use op-scim-bridge
    ```
 
-   Copy the example command to a text editor. Replace `user` with the appropriate username for your Linux server and `scim.example.com` with the host name or IP address used for SSH access to the Linux server before running the above commands in your terminal.
+   Copy the example command to a text editor. Replace `user` with the appropriate username for your Linux server and `scim.example.com` with the host name or IP address used for SSH access to the Linux server before running the command in your terminal.
 
-5. Initialize the Swarm. Run this command:
+   > **Note**
+   >
+   > 🔑 You can store your SSH keys in 1Password and authenticate this and other SSH workflows without writing a
+   > private key to disk using the [1Password SSH agent](https://developer.1password.com/docs/ssh/get-started).
+
+6. Switch to the Docker context you just created to connect to the Docker host:
+
+   ```sh
+   docker context use op-scim-bridge
+   ```
+
+7. Initialize the Swarm. Run this command:
 
    ```sh
    docker swarm init # --advertise-addr 192.0.2.1
    ```
 
    > **Note**
    >
-   > If Docker returns an error about multiple network interfaces, uncomment the `--advertise-addr` parameter (delete `#`) and replace the example IP (`192.0.2.1`) with the IP address of a network interface on which the server can be reached externally.
+   > If Docker returns an error about multiple network interfaces, uncomment the `--advertise-addr` parameter (delete
+   > `#`) and replace the example IP (`192.0.2.1`) with the IP address of a network interface on which the server can
```

Comment:
Typically when I've done this the default network interface will have both a private and public IP. I always sit and scratch my dumb head for a moment trying to think about which one to use here. Can we state explicitly which IP should be used or provide some guidance ("choose the external IP address") or whatever makes sense?

Reply:
I punted to the Docker Swarm docs for this in the very next lines, but perhaps I can clarify the explanation here a bit and expand the link text.

Reply:
How is [this](https://github.com/1Password/scim-examples/pull/252/commits/a75f3ef93873f3763b5f26349a525a9a3fa8e955)?

Reply:
I don't know if the Docker docs nor your changes necessarily help me pick one of the two IP addresses assigned to the primary network interface (in my experience there are always two), but I might be an edge-case on account of being very prone to overthinking so this is probably fine.

Reply:
Thanks. Maybe I'll take another whack at it just the same. 👍

Reply:
I've expanded it significantly in [this commit](https://github.com/1Password/scim-examples/pull/252/commits/1a775b60b8e80064df16569dfee12856d69c5dae).

It's moot for the generic deployment example anyway, since we're recommending a single node. It just needs to be an IP address that another node can use to reach this one.